src/ApplicationResolver.php

```diff
@@ -34,7 +30,15 @@ final class ApplicationResolver
      */
     public static function resolve(): Application
     {
-        $app = (new self)->createApplication();
+        if (file_exists(getcwd().'/testbench.yaml')) {
+            $appBasePath = transform(Yaml::parseFile(getcwd().'/testbench.yaml'), function ($config) {
+                return $config['laravel'];
+            });
+        }
+
+        $appBasePath ??= Testbench::applicationBasePath();
```

Comment:
This could be so simple.
```suggestion
        $appBasePath = file_exists(getcwd().'/testbench.yaml')
            ? transform(Yaml::parseFile(getcwd().'/testbench.yaml'), function ($config) {
                return $config['laravel'];
            })
            : Testbench::applicationBasePath();
```